front/public/locales/en/errors.json

```diff
@@ -153,6 +153,12 @@
     "stdcm": {
       "InfraNotFound": "Infra {{infra_id}} does not exist"
     },
+    "stdcm_v2": {
+      "InfraNotFound": "Infra {{infra_id}} does not exist",
+      "InvalidPathItem": "Path item {{index}} is invalid",
+      "RollingStockNotFound": "Rolling stock {{rolling_stock_id} does not exist",
+      "TimetableNotFound": "Timetable {{timetable_id}} does not exist"
```

Comment:
On other erros (like l.163), the variable is surrounded with apostrophes. Would it be possible to define a convention for this ?

Reply:
We should always use apostrophes.